src/Compilers/Core/CodeAnalysisTest/CorLibTypesTests.cs

```diff
@@ -96,7 +96,6 @@ public void ConstantValueInvalidOperationTest01()
 
             var cv = ConstantValue.Create(1);
             Assert.Throws<InvalidOperationException>(() => { var c = cv.StringValue; });
-            Assert.Throws<InvalidOperationException>(() => { var c = cv.CharValue; });
```

Comment:
>Assert.Throws<InvalidOperationException>(() => { var c = cv.CharValue; }); [](http://example.com/codeflow?start=12&length=74)

Let's test this scenario and adjust expectations accordingly, i.e. assert what ```cv.CharValue``` returns.